.github/workflows/copy-release-assets.yml

```diff
@@ -4,28 +4,46 @@ on:
   release:
     types: published
 
-permissions:
-  contents: write
-
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
   release:
-    name: Copy assets to the new release
+    name: copy-release-assets
```

Comment:
> Why this change?

This isn't the name that you see in the workflow panel i.e.

![image](https://github.com/tldr-pages/tldr/assets/26346867/fa0ad63f-3f92-4ce1-8a49-8fe8b6c2018d)

But rather the one used for the job, the current one currently is a bit long so the contents are cut (when viewing the job), so I changed it so that the entire word fits the screen.

![image](https://github.com/tldr-pages/tldr/assets/26346867/2b59d252-2dfb-4814-9f9f-3b6d87ac3f78)